addons/web_tour/static/src/widgets/tour_start.js

```diff
@@ -0,0 +1,49 @@
+import { charField, CharField } from "@web/views/fields/char/char_field";
+import { useService } from "@web/core/utils/hooks";
+import { registry } from "@web/core/registry";
+
+export class TourStartWidget extends CharField {
+    static template = "web_tour.TourStartWidget";
+    static props = {
+        ...CharField.props,
+        icon: { type: Boolean, optional: true },
+    };
+
+    setup() {
+        const { data } = this.props.record;
```

Comment:
wrong. in this case the Object data is fixed in the component for the whole component's life cycle.
make a getter:
```js
get tourData() {
   return this.props.record.data
}

 ```